testdata/project-v3/config/manager/manager.yaml

```diff
@@ -33,6 +33,14 @@ spec:
         name: manager
         securityContext:
           allowPrivilegeEscalation: false
+        livenessProbe:
```

Comment:
wdyt about we add for the livenessProbe:

initialDelaySeconds: 5
      periodSeconds: 10